README.md

```diff
@@ -57,9 +57,3 @@ FunnyGuilds hooks into those plugins:
 
 #### Stats
 Current statistics (since 3.9.x) - [BStats](https://bstats.org/plugin/bukkit/FunnyGuilds)
-<br>
-The best reported statistics (on [MCStats](http://mcstats.org/plugin/FunnyGuilds)):
-
-| Servers | Players | Users     | Guilds |
-|---------|---------|-----------|--------|
-| 1158    | 12 336  | 1 034 312 | 16 973 |
```

Comment:
Ja bym zostawił te najlepsze staty, i ewentualnie dopisał że one na MCStats - w końcu pochwalić się trzeba, a że nie działa no to cóż